release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -2397,6 +2397,8 @@ Identify the disk that holds the live image, in this example `nvme0n1p3`, and ru
 
 * There is currently a known issue with OVN-Kubernetes that whenever the `NetworkManager` service restarts the node will lose network connectivity and must be recovered. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2074009[*BZ#2074009*])
 
+* The default security context constraint (SCC) might cause a pod using generic ephemeral volumes to remain in the pending state. To work around this issue, you can create a custom SCC. For more information, see link:https://access.redhat.com/solutions/6967807[*KCS*] , and for the workaround, see link:https://access.redhat.com/articles/6967808[*KCS*]. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2100429[*BZ#2100429*])
```

Comment:
Revision suggestion: 

* The default security context constraint (SCC) might cause a pod using generic ephemeral volumes to remain in the `Pending` state. To work around this issue, you can create a custom SCC. For more information, see link:https://access.redhat.com/solutions/6967807[Pods with Generic Ephemeral Volumes fail with SCC errors]. For the workaround, see link:https://access.redhat.com/articles/6967808[Allowing the use of generic ephemeral volumes
]. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2100429[*BZ#2100429*])